lambda/custom/state_handlers.js

```diff
@@ -377,6 +399,74 @@ var scout_agent = (function() {
             reject('Scout Unavailable');
           });
       });
+    },
+    getArticleStatus: function(userId, articleId) {
+      return new Promise((resolve, reject) => {
+        console.log('getArticleStatus: ' + articleId + ' for ' + userId);
```

Comment:
Consider template literals for all of these string constructions -- they can be easier to read (and easier to type!):
```
console.log(`getArticleStatus: ${articleId} for ${userId}`);
```